src/readcalls.c

```diff
@@ -185,7 +183,7 @@ int readcalls(const char *logfile) {
 	exit(1);
     }
 
-    while (fgets(inputbuffer, LOGLINELEN + 1, fp) != NULL) {
+    while (fgets(inputbuffer, sizeof(inputbuffer), fp) != NULL) {
```

Comment:
Just cosmetics (and trying to have a single point of definition). Could also change the next line too.